packages/astro/test/fixtures/content-layer/src/content/config.ts

```diff
@@ -86,6 +85,21 @@ const spacecraft = defineCollection({
 		}),
 });
 
+const reptiles = defineCollection({
+	type: 'experimental_content',
+	loader: glob({
+		pattern: '*.{mdx, md}',
+		base: new URL('../../content-outside-src-mdx', import.meta.url),
```

Comment:
```suggestion
		base: './content-outside-src-mdx'
```
Base can be relative to the site root